crates/bevy_pbr/src/light.rs

```diff
@@ -359,6 +359,10 @@ pub fn update_clusters(windows: Res<Windows>, mut views: Query<(&Camera, &mut Cl
         let inverse_projection = camera.projection_matrix.inverse();
         let window = windows.get(camera.window).unwrap();
         let screen_size_u32 = UVec2::new(window.physical_width(), window.physical_height());
+        // Don't update clusters if screen size is 0.
+        if screen_size_u32 == UVec2::ZERO {
+            return;
```

Comment:
```suggestion
            continue;
```

It would allow to work when having multiple windows